app/pages/notifications/moderation.cjsx

```diff
@@ -0,0 +1,84 @@
+React = require 'react'
+{Link} = require '@edpaget/react-router'
+{Markdown} = require 'markdownz'
+moment = require 'moment'
+talkClient = require '../../api/talk'
+apiClient = require '../../api/client'
+Loading = require '../../components/loading-indicator'
+Avatar = require '../../partials/avatar'
+
+module?.exports = React.createClass
+  displayName: 'ModerationNotification'
+
+  propTypes:
+    project: React.PropTypes.object
+    user: React.PropTypes.object.isRequired
+    notification: React.PropTypes.object.isRequired
+
+  getInitialState: ->
+    moderation: null
+    comment: null
+    commentUser: null
+    reports: []
+
+  componentWillMount: ->
+    talkClient.type('moderations').get(@props.notification.source_id).then (moderation) =>
+      comment = moderation.target or moderation.destroyed_target
+      @setState {moderation, comment}
+
+      promises = []
+      for report in moderation.reports then do (report) =>
+        promises.push apiClient.type('users').get(report.user_id.toString(), { }).then (user) =>
```

Comment:
Since a destroyed comment is just an embedded JSON copy of what the comment was, it doesn't get formatted by the serializer -- the id's are going to be integers